doc/python/axes.md

```diff
@@ -154,6 +154,25 @@ fig.update_yaxes(ticklabelposition="inside top", title=None)
 fig.show()
 ```
 
+#### Specifying Label Aliases 
+
+*New in 5.14*
+
+With `labelalias`, you can specify replacement text for specific tick and hover labels. In this example, the dataset has the values of "Sat" and "Sun" in the day column. By setting `labelalias=dict(Sat="Saturday", Sun="Sunday")`, we swap these out for "Saturday" and "Sunday".
+
+```python
+import plotly.express as px
+import pandas as pd
+
+df = px.data.tips()
+df = df[df.day.isin(['Sat', 'Sun'])].groupby(by='day', as_index=False).sum(numeric_only=True)
+
+fig = px.bar(df, x="day", y="total_bill")
+fig.update_xaxes(labelalias=dict(Sat="Saturday", Sun="Sunday"))
+
+fig.show()
+```
+
```

Comment:
Thanks @archmoj , is the distinction between using this on the `colorbar` or `ticktext` that with `ticktext` you'd need to provide a new value for all values?
I'm looking at this existing example:
https://plotly.com/python/colorscales/#custom-color-bar-title-labels-and-ticks-with-graph-objects

If we wanted to just show the "Hot" and "Cold" but the number value for "Mild", we could add `labelalias` like this
```
labelalias={100: "Hot", 2: "Cold"},
```
Instead of 
```
ticktext=["Cool", "Mild", "Hot"],
```

Is that the idea?

Reply:
Yes. That's the idea.

Reply:
Added an example on the colorscales page